docs/src/content/en/docs/workflows/human-in-the-loop.mdx

```diff
@@ -1,268 +1,126 @@
-      `,
-      {
-        structuredOutput: {
-          schema: z.object({
-            response: z.string(),
-            gameWon: z.boolean(),
-          }),
-        },
-      },
-    );
-
-    const { response: agentResponse, gameWon } = response.object;
-
-    guessCount++;
+    await fetch("https://httpbin.org/post", {
+      method: "POST",
+      body: JSON.stringify({ to: userEmail })
+    });
 
-    return { famousPerson, gameWon, agentResponse, guessCount };
-  },
+    return {
+      output: `Email sent to ${userEmail}`
+    };
+  }
 });
 
-const winStep = createStep({
-  id: "win-step",
-  description: "Handle game win logic",
+export const testWorkflow = createWorkflow({
+  id: "test-workflow",
   inputSchema: z.object({
-    famousPerson: z.string(),
-    gameWon: z.boolean(),
-    agentResponse: z.string(),
-    guessCount: z.number(),
+    userEmail: z.string()
   }),
   outputSchema: z.object({
-    famousPerson: z.string(),
-    gameWon: z.boolean(),
-    guessCount: z.number(),
-  }),
-  execute: async ({ inputData }) => {
-    const { famousPerson, gameWon, guessCount } = inputData;
+    output: z.string()
+  })
+})
+  .then(step1)
+  .commit();
+```
 
-    console.log("famousPerson: ", famousPerson);
-    console.log("gameWon: ", gameWon);
-    console.log("guessCount: ", guessCount);
+## Applying human input with `resume()`
 
-    return { famousPerson, gameWon, guessCount };
-  },
+Use `resume()` to restart a suspended workflow from the step where it paused. To satisfy the step’s suspend condition, pass a value using `resumeData` that matches the step's `resumeSchema`, allowing execution to continue from where it left off.
+
+![Restarting a workflow with resume()](/img/workflows/workflows-suspend-resume-resume.jpg)
+
+```typescript {13} showLineNumbers copy
+const workflow = mastra.getWorkflow("testWorkflow");
+const run = await workflow.createRunAsync();
+
+const result = await run.start({
+  inputData: {
+    userEmail: "alex@example.com"
+  }
 });
 
-export const headsUpWorkflow = createWorkflow({
-  id: "heads-up-workflow",
-  inputSchema: z.object({
-    start: z.boolean(),
-  }),
-  outputSchema: z.object({
-    famousPerson: z.string(),
-    gameWon: z.boolean(),
-    guessCount: z.number(),
-  }),
-})
-  .then(startStep)
-  .dountil(gameStep, async ({ inputData: { gameWon } }) => gameWon)
-  .then(winStep)
-  .commit();
+const handleResume = async () => {
+  const resumedResult = await run.resume({
+    step: "step-1",
+    resumeData: { okToSend: true }
+  });
+  console.log(resumedResult);
+};
 ```
 
-> See [Workflow](/reference/workflows/workflow) for a full list of configuration options.
+## Surfacing suspended reasons
 
-## Registering the agents and workflow
+When a workflow is suspended, the step can return structured data that explains why it was paused and what input is needed to resume. The data is returned using `suspend()` and available in the `suspendPayload`.
```

Comment:
this reads a lot like a **Reference**

> When a workflow is suspended, the step can return structured data that explains why it was paused and what input is needed to resume.

under **Docs**, I would expect to learn _when_ to use this